projects/plugins/jetpack/class-jetpack-newsletter-dashboard-widget.php

```diff
@@ -100,33 +118,47 @@ public static function load_admin_scripts( $asset_handle, $asset_name, $options
 		$default_options = array(
 			'config_data'          => array(),
 			'config_variable_name' => 'configData',
-			'enqueue_css'          => true,
 		);
 		$options         = wp_parse_args( $options, $default_options );
-		if ( file_exists( __DIR__ . "/../dist/{$asset_name}.js" ) ) {
+		if ( file_exists( __DIR__ . "/../dist/{$asset_name}.min.js" ) ) {
 			// Load local assets for the convinience of development.
 			Assets::register_script(
 				$asset_handle,
-				"../dist/{$asset_name}.js",
+				"../dist/{$asset_name}.min.js",
 				__FILE__,
 				array(
 					'in_footer'  => true,
 					'textdomain' => 'jetpack',
+					'css_path'   => "../dist/{$asset_name}.css",
 				)
 			);
 			Assets::enqueue_script( $asset_handle );
 		} else {
 			// In production, we load the assets from our CDN.
 			wp_register_script(
 				$asset_handle,
-				sprintf( self::NEWSLETTER_WIDGET_CDN_URL, "{$asset_name}.js" ),
+				sprintf( self::NEWSLETTER_WIDGET_CDN_URL, "{$asset_name}.min.js" ),
 				self::JS_DEPENDENCIES,
 				self::NEWSLETTER_WIDGET_VERSION,
 				true
 			);
 			wp_enqueue_script( $asset_handle );
 		}
 
-		// TODO: Add config data.
+		$css_url    = $asset_name . ( is_rtl() ? '.rtl' : '' ) . '.css';
+		$css_handle = $asset_handle . '-style';
+		wp_register_style(
```

Comment:
Should this be inside the check for `file_exists( __DIR__ . "/../dist/{$asset_name}.min.js" )`, since locally we already load the CSS locally via `css_path`?